types/pump/pump-tests.ts

```diff
@@ -52,3 +53,16 @@ pump(createReadStream('/dev/random'), toHex(), createWriteStream('/dev/null'));
 
 // $ExpectType Stream
 pump([createReadStream('/dev/random'), toHex(), createWriteStream('/dev/null')]);
+
+const copy: Gulp.TaskFunction = (cb) => {
+    pump([
+        Gulp.src('/dev/random'),
+        Gulp.dest('/dev/null'),
+    ], (err) => {
+        if (err) {
+            cb(err);
+            return;
+        }
+        console.log('Copy done with err = ' + err);
```

Comment:
Don't put console logs in tests. You might want to review what exactly the test files in definitely typed are for, my friend. 

https://github.com/DefinitelyTyped/DefinitelyTyped#my-package-teststs

Reply:
I was under the impression that console logs weren't actually being executed, but I'm aware it appears somewhat contrived so I've replaced it with something else 8f1cf7a.